apps/explorer/components/SNIDetails.vue

```diff
@@ -7,15 +7,19 @@
       <h1 class="mb-4 text-4xl">{{ detail.name }}</h1>
       <ul>
         <li v-for="(value, key) in detail" :key="key" class="my-2">
-          <div
-            v-if="key !== 'image' && key !== 'createdAt' && key !== 'updatedAt'"
-          >
-            {{ splitCamelCase(key.toString()) }}:
-            {{ value ? truncate(value, 30) : '-' }}
-          </div>
-          <div v-if="key === 'createdAt' || key === 'updatedAt'">
-            {{ splitCamelCase(key.toString()) }}:
-            {{ convertToSimpleFormat(+value!) }}
+          <div v-if="key !== 'image'">
+            <span> {{ splitCamelCase(key.toString()) }}: </span>
+            <a
+              v-if="key === 'owner'"
+              :href="`https://moonscan.io/address/${value}`"
```

Comment:
The network should be set in config. Right now we are on testnet, so we are using moonbase.